package.json

```diff
@@ -20,7 +20,7 @@
     "gulp-uglify": "^1.5.3",
     "knuth-shuffle": "^1.0.1",
     "testem": "^1.0.2",
-    "typescript": "^2.0.6",
+    "typescript": "^2.1.0",
```

Comment:
Clarifying, because of using `keyof`, snabbdom users and frameworks still based on TypeScript v2.0 *won't* be able to compile their projects if this PR is in snabbdom.